JS-Algorithms/challenges.js

```diff
@@ -118,7 +137,17 @@ Because the 12th index in the Fibonacci sequence is 144, and 144 has three digit
 ***** */
 
 const fibIndex = (n) => {
-  // YOUR CODE HERE...
+   let n1 = 0,
+      n2 = 1,
+      nextTerm = n1 + n2;
+   let fiboArray = [n1, n2, nextTerm];
```

Comment:
If you don't mutate this variable use `const` instead of `let`

Reply:
This variable could ve a const instead of a let 
 const fiboArray = [n1, n2, nextTerm];

Thank you so much for all your comments, appreciate it, have a nice day :+1: